nautobot/extras/models/groups.py

```diff
@@ -234,13 +394,12 @@ def generate_filter_form(self):
 
         This form is used to popoulate and validate the filter dictionary.
         """
-        filterform_class = self.map and self.map.filterform_class
```

Comment:
Do we need here to either call `self._set_object_classes` or otherwise check that `self.filterform_class is not None`?

Reply:
Nah, it'll just return `None` if it's not valid. This is an expected use-case for the UI such as when a new model instance is created in "add" view. We don't want there to be a filter form generated at render time on a new instance where `content_type` isn't yet selected. I'll expand the docstring.